extensions/sql-database-projects/src/models/project.ts

```diff
@@ -466,17 +479,19 @@ export class Project {
 		return found;
 	}
 
-	private addDatabaseReferenceChildren(referenceNode: any, name?: string): void {
-		let suppressMissingDependenciesErrorNode = this.projFileXmlDoc.createElement(constants.SuppressMissingDependenciesErrors);
-		let falseTextNode = this.projFileXmlDoc.createTextNode('False');
+	private addDatabaseReferenceChildren(referenceNode: any, entry: IDatabaseReferenceProjectEntry): void {
+		// TODO: create checkbox for this setting
+		const suppressMissingDependenciesErrorNode = this.projFileXmlDoc.createElement(constants.SuppressMissingDependenciesErrors);
+		const falseTextNode = this.projFileXmlDoc.createTextNode('False');
 		suppressMissingDependenciesErrorNode.appendChild(falseTextNode);
 		referenceNode.appendChild(suppressMissingDependenciesErrorNode);
 
-		if (name) {
-			let databaseVariableLiteralValue = this.projFileXmlDoc.createElement(constants.DatabaseVariableLiteralValue);
-			let databaseTextNode = this.projFileXmlDoc.createTextNode(name);
-			databaseVariableLiteralValue.appendChild(databaseTextNode);
-			referenceNode.appendChild(databaseVariableLiteralValue);
+		// TODO: add support for sqlcmd vars and server
```

Comment:
Can you create a GH issue for this and add the URL here?

Reply:
added url for https://github.com/microsoft/azuredatastudio/issues/12036